ksqldb-execution/src/main/java/io/confluent/ksql/execution/builder/KsqlQueryBuilder.java

```diff
@@ -207,7 +217,18 @@ public KsqlQueryBuilder withKsqlConfig(final KsqlConfig newConfig) {
         ksqlConfig,
         serviceContext.getSchemaRegistryClientFactory(),
         loggerNamePrefix,
-        processingLogContext
+        processingLogContext,
+        getSerdeTracker(loggerNamePrefix)
+    );
+  }
+
+  private Optional<TrackedCallback> getSerdeTracker(final String loggerNamePrefix) {
+    if (System.getProperty(KSQL_TEST_TRACK_SERDE_TOPICS) == null) {
```

Comment:
nit: worth checking equality to `true` in case people think they can just flip it from "true"/"false"

Reply:
Meh, it's not something anyone should be setting every, except in QTT, so I don't see it as an issue.